foreman-proxy/foreman-proxy.init

```diff
@@ -58,6 +58,20 @@ stop() {
     return $RETVAL
 }
 
+logrotate() {
+    echo -n $"Rotating logs for $prog: "
+    if [ -f ${FOREMAN_PROXY_PID} ]; then
+        killproc -p ${FOREMAN_PROXY_PID} -USR1
```

Comment:
This is missing an argument between the pid file and `-USR1`, as currently it's issuing a SIGTERM every time.  Adding -x shows `+ base=-USR1`, but if you instead run:

    killproc -p /var/run/foreman-proxy/foreman-proxy.pid foreman-proxy -USR1

then it logs:

    + killlevel=-USR1
    + base=foreman-proxy

and the process gets rotated.